app/containers/WeatherContainer/index.js

```diff
@@ -0,0 +1,182 @@
+/**
+ *
+ * WeatherContainer
+ *
+ */
+
+import React, { useEffect, memo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl, FormattedMessage as T } from 'react-intl';
+import Clickable from '@components/Clickable';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { useInjectSaga } from '@utils/injectSaga';
+import styled from 'styled-components';
+import debounce from 'lodash/debounce';
+import isEmpty from 'lodash/isEmpty';
+import get from 'lodash/get';
+import { Card, Input, message, Button, Space } from 'antd';
+import { selectWeatherContainer, selectCityData, selectCityError, selectCityName } from './selectors';
+import { weatherContainerCreators } from './reducer';
+import saga from './saga';
+
+const { Search } = Input;
+
+const CustomCard = styled(Card)`
+  && {
+    margin: 20px 0;
+    max-width: ${props => props.maxwidth};
+    color: ${props => props.color};
+    ${props => props.color && `color: ${props.color}`};
+  }
+`;
+const DetailsCard = styled(Card)`
+  margin: 20px 20px;
+`;
+const Container = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    max-width: ${props => props.maxwidth}px;
+    width: 100%;
+    margin: 0 auto;
+    padding: ${props => props.padding}px;
+  }
+`;
+const CustomDiv = styled.div`
+  width: 100%;
+  font-weight: bolder;
+  font-size: 20px;
+`;
+const RightContent = styled.div`
+  display: flex;
+  align-self: flex-end;
+`;
+const CustomSpace = styled(Space)`
+  margin: 25px;
+`;
+export function WeatherContainer({ dispatchCityWeather, cityData = {}, cityError = null, cityName, intl }) {
```

Comment:
this is a mistake in the react-template as well, the better way to do it is by using defaultProps.